### ISSUE TITLE
Fallback to clipboard if nothing is selected

### ISSUE THREAD

**Issue by jonas-w** [MassimilianoPasquini97/raycast_ollama#4]
Thanks for this neat extension.

My problem with most of the predefined 'tasks' is that you need to have something selected.

As I spend most of my time in the terminal in the [helix-editor](https://github.com/helix-editor/helix) it does not recognize it as a selection, which is expected. That's why I would prefer it, if it can't detect anything selected, that it falls back to using the clipboard content.

This could also be a configurable setting, so it doesn't need to check the selection first but immediately uses the clipboard content.

**Comment by MassimilianoPasquini97** [comment]
Hi @jonas-w, 
I have created a new preference "***Input Source***" which allows to chose preferred input method between 'Selected Text' and 'Clipboard'. In case text is empty on preferred method it fallback to the other one.

I glad if you could clone the [clipboard_fallback](https://github.com/MassimilianoPasquini97/raycast_ollama/tree/clipboard_fallback) branch and try it and leet me know if further improvement is necessary.

**Comment by MassimilianoPasquini97** [comment]
I have added a new preference named 'Enable Input Source Fallback' https://github.com/MassimilianoPasquini97/raycast_ollama/commit/da0deabab9f305c43754319fcb2b4c5db4185d6d, disabled by default for enable fallback behavior. 

Thanks @jonas-w for your feedback and suggestion, we have improved this extension. If there isn't any other improvement to suggest we can close this issue and merge to the main branch.
